public/images/feather/heart.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-heart"><path d="M20.84 4.61a5.5 5.5 0 0 0-7.78 0L12 5.67l-1.06-1.06a5.5 5.5 0 0 0-7.78 7.78l1.06 1.06L12 21.23l7.78-7.78 1.06-1.06a5.5 5.5 0 0 0 0-7.78z"></path></svg>
```

Comment:
We can discuss about folder hierarchy, here, I created `/public/images/feather folder` for this icon only.

Reply:
I have no strong opinion about the folder name. Mentioning "feather" may not be necessary unless we plan to use these icons specifically in the future.

I am a bit reluctant to store these icons in the "public" folder though. Other "public" files are compressed in the Docker image, to be served directly via an explicit HTTP request. But these icons work differently. They are bundled in the .js files at build time. So in my opinion, it would be clearer to put them in "src".
And that would help to keep in mind these files behave like other .js dependencies, eg they should be bundled separately and loaded lazily if they become too heavy.

Reply:
I agree with all your points. Storing svg inside /public folder would only make sense if we need them to be publicly accessible.
So, what about storing all new svg icons inside `src/icons` ?

Reply:
I agree we don't need to mention the origin set, `feather` here, in the folder, it may be more confusing than helpful.

About where icons are stored, I wouldn't mind if they where in `/public/`, ready to be used as real images if needed like @xem says. But there is no use case right now, at least in this PR, so ok to move on with `/src/icons` here.
And in anycase, thanks to the file responsible for the imports, it's completely possible to have a mix of imports of SVG files from `public` and `src` if we need that in the future.